rust_src/src/lib.rs

```diff
@@ -126,6 +126,7 @@ mod windows;
 mod xdisp;
 mod xfaces;
 mod xml;
+mod xsettings;
```

Comment:
I think you can drop the `#[cfg(feature = "window-system-x11")]` in this file and just mark the file with it instead.